backoffice/backoffice-domain-service/ballerina/modules/backoffice/resources/backoffice-api.yaml

```diff
@@ -3519,7 +3519,7 @@ components:
         type: string
         enum:
           - api
-          - subcription
+          - subscription
```

Comment:
Shall we not change this for the moment? Not sure whether we might have to check for any code level changes for this

Reply:
Fixed in https://github.com/wso2/apk/pull/864/commits/d0d0feff6fdfdd198dc75322a3ace6e900743e89